assets/translate.js

```diff
@@ -128,6 +138,7 @@ $( window ).on( 'load', function () {
 					// Otherwise, it will needlessly update when the input is being enabled
 					return;
 				}
+				window.alreadyUpdating = true;
```

Comment:
does this need to be in the global scope? looks like we're only using it within this function, so it could go in the `.on('load', function () {` scope, maybe (since you probably don't want it redefined for each `.oo-ui-inputWidget`)